packages/protocol/test/signal/SignalService.t.sol

```diff
@@ -2,6 +2,7 @@
 pragma solidity 0.8.24;
 
 import "../TaikoTest.sol";
+import "forge-std/src/console2.sol";
```

Comment:
@cyberhorsey I added your test in another PR: [add_jeff_test](https://github.com/taikoxyz/taiko-mono/pull/16090)